packages/dotcom-ui-flags/src/server/formatFlagsJSON.ts

```diff
@@ -1,6 +1,6 @@
 import { TFlagsData } from '../types'
 
-export default function formatFlagsJSON(flags: TFlagsData = {}): string {
+export default function formatFlagsJSON(flags: TFlagsData = {}): object {
```

Comment:
Thinking out loud, but I'm not sure this function is doing what it says anymore. If we're not formatting the list of flags as a JSON string anymore then this is filerring out the false flags rather than formatting them (which can be done much more cleanly than this if we're working with objects now!)

Reply:
Will see if can make this better thanks

Reply:
Looking into this, the functionality still makes sense the main difference is I've moved when this data gets stringified to match AppContext so that they can be refactored to use DataEmbed.